specifyweb/frontend/js_src/lib/components/Toolbar/Query.tsx

```diff
@@ -125,101 +124,90 @@ export function QueryListDialog({
 
   const isReadOnly = React.useContext(ReadOnlyContext);
 
-  return data === undefined ? (
-    <Dialog
```

Comment:
isn't this still needed for the other places where QueryListDialog is used though?
so instead of removing it, could you instead move it into the `defaultChildren` function?

Reply:
I added the skeleton in the default children but not sure that's what you meant

Reply:
yes, that's what I meant - thanks

this way, by default, consumers of this component still get a loader, but they don't get a bug if they choose to provide a custom `children` function